catt/controllers.py

```diff
@@ -54,15 +86,23 @@ def clear(self):
 
 
 class CastController:
-    def __init__(self):
+    def __init__(self, device_name):
         cache = Cache()
+        cached_ip = cache.get(device_name)
```

Comment:
Yes, we do need to always be looking up by `get_chromecasts()` and selecting the closest one, but we only have to select the first matching one (no complicated logic necessary). Either way, I'm not really hung up on this, so let's just leave it as-is for now.